packages/drivers/odsp-driver/src/contractsPublic.ts

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IOdspUrlParts } from "@fluidframework/odsp-driver-definitions";
+
+/**
+ * @deprecated - use OdspFluidDataStoreLocator
+ */
+export type OdspDocumentInfo = OdspFluidDataStoreLocator;
+
+export interface OdspFluidDataStoreLocator extends IOdspUrlParts {
+    dataStorePath: string;
+    appName?: string;
+    containerPackageName?: string;
+    fileVersion?: string;
+}
+
+export enum SharingLinkHeader {
+    // Can be used in request made to resolver, to tell the resolver that the passed in URL is a sharing link
```

Comment:
Should we also move this to defn package?

Reply:
The initial desire with the package I've created is to isolate all the definitions that are required for driver factory, as that's the part that we want to dynamically load (with older versions of loader).
Everything else is bundles with app and thus has same version as loader.
So the answer is no, but it shows that the name of the package is not really reflecting what it is.
At the same time, I think that's fine, because all the other stuff (i.e. not factory) likely needs to be moved out to some other package.